src/main/kotlin/sliv/tool/parsers/points/CSVPointsParser.kt

```diff
@@ -0,0 +1,6 @@
+package sliv.tool.parsers.points
+
+import sliv.tool.models.landmarks.csv.CSVPointLandmark
+import sliv.tool.parsers.CSVParser
+
+object CSVPointsParser : CSVParser<CSVPointLandmark>(CSVPointLandmark::class.java)
```

Comment:
now we always return raw data classes? what about mappers to project and visualization models?